Consultant/Consultant.md

```diff
@@ -26,25 +26,25 @@
   ```bash
   {
     "**Prompt:**": "Can you advice me on preventive care for my pet?",
-    "**Response:**": "Absolutely! Preventive care for your pet includes regular vaccinations, maintaining a balanced diet, ensuring they get enough exercise, and scheduling routine check-ups. For a personalized preventive care plan, I recommend consulting with a vet through Petofy's consultation service"
+    "**Response:**": "Absolutely! Preventive care for your pet includes regular vaccinations, maintaining a balanced diet, ensuring they get enough exercise, and scheduling routine check-ups. For a personalized preventive care plan, I recommend consulting with a vet through https://petofy.com/VetOnCall."
   }
   ```
   ```bash
   {
     "**Prompt:**": "What type of diet is best for my pet's breed and age?",
-    "**Response:**": "The ideal diet depends on your pet's breed, age, and overall health. I recommend consulting with a vet through Petofy's vet on-call service. They can provide expert advice to ensure your pet gets the nutrition it requires."
+    "**Response:**": "The ideal diet depends on your pet's breed, age, and overall health. I recommend consulting with a vet through https://petofy.com/VetOnCall. They can provide expert advice to ensure your pet gets the nutrition it requires."
   }
   ```
   ```bash
   {
     "**Prompt**": "My pet is unconscious, and it looks like it ate something strange. What should I do?",
-    "**Response**": "Contact your vet immediately if your pet ingests something toxic. Provide details on the substance and follow their guidance for prompt and appropriate care."
+    "**Response**": "Contact your vet immediately. You can also use our vet on call service at https://petofy.com/VetOnCall. Provide details on the substance, and follow their guidance for prompt and appropriate care."
```

Comment:
@ANMOL2001A See if this can be discussed with Petofy team and take their suggestion regarding adding phone number

Reply:
@pandeytapan Sure sir if we get direct vets number i'll add that